### ISSUE TITLE
replace native notification for "copied to clipboard" message

### ISSUE THREAD

**Issue by adamwatters** [adamwatters/vscode-color-pick#10]
instead of popping up a notification for each color change, show a single message somewhere in the webview and just update the color string. maybe include an animation.

**Comment by JacksonKearl** [comment]
Would you accept a PR for this? Alternatively an option to disable the notifications. Currently its impossible to type in the picker boxes as the notifications steal focus.

**Comment by adamwatters** [comment]
yes totally! sorry for the delay - I haven't worked on this project in a while!

**Comment by adamwatters** [comment]
Sorry Jackson - I'm trying to get this change merged, but I can't even build my own project!... 😮‍💨
